charts/rd-location-ref-api/Chart.yaml

```diff
@@ -2,4 +2,11 @@ apiVersion: v2
 appVersion: "1.0"
 description: A Helm chart for rd-location-ref-api
 name: rd-location-ref-api
-version: 0.0.11
\ No newline at end of file
+home: https://github.com/hmcts/rd-location-ref-api
+version: 0.0.11
+maintainers:
+  - name: Reference Data Team
+dependencies:
+  - name: java
+    version: ~3.4.4
+    repository: '@hmctspublic'
```

Comment:
dont u need to add apiVersion: v2??

Reply:
It is already there in line no.1 - https://github.com/hmcts/rd-location-ref-api/blob/0769d8e839630c6a04dd8f2b2b038734d6dafed5/charts/rd-location-ref-api/Chart.yaml#L1

Reply:
👍